### PR TITLE
fix display of GOR in realtime mode

### PR DESCRIPTION
**Pull request by emlynjdavies**
GOR value in live raw within the GUI is now the same as in interactive summary

### CODE REVIEW

**Comment by emlynjdavies** [comment]
fixes #114

File: pysilcam/silcamgui/guicalcs.py [hunk 1]
```diff
@@ -434,54 +426,58 @@ def plot(self):
...
+                    'Gas d50: {:0.0f} [um]'.format(gas_d50) + '\n' +
+                    'GOR: {:0.2f}'.format(gor) + ' ' + ' Saturation: {:0.0f} [%]'.format(saturation)
+            )
+            plt.title(ttlstr)
+            plt.imshow(imraw)
+            plt.axis('off')
+
+            plt.subplot(1, 2, 2)
+            ttlstr = ('Image time: ' +
+                        str(timestamp))
```
Comment (tankefugl) [inline pysilcam/silcamgui/guicalcs.py:474]: E127: continuation line over-indented for visual indent

File: pysilcam/silcamgui/guicalcs.py [hunk 2]
```diff
@@ -434,54 +426,58 @@ def plot(self):
...
-                plt.axis('off')
-                plt.title(ttlstr)
-
-                plt.tight_layout()
+
+            try:
+                guidata = self.q.get(timeout=0.1)
+            except:
+                return
+
```
Comment (tankefugl) [inline pysilcam/silcamgui/guicalcs.py:434]: W293: blank line contains whitespace

File: pysilcam/silcamgui/guicalcs.py [hunk 3]
```diff
@@ -434,54 +426,58 @@ def plot(self):
...
-                plt.cla()
-                plt.imshow(imc)
-                plt.axis('off')
-                plt.title(ttlstr)
-
-                plt.tight_layout()
+
+            try:
+                guidata = self.q.get(timeout=0.1)
+            except:
```
Comment (tankefugl) [inline pysilcam/silcamgui/guicalcs.py:432]: E722: do not use bare 'except'

File: pysilcam/oilgas.py [hunk 4]
```diff
@@ -142,10 +143,21 @@ def update(self):
...
         self.dias, self.vd_gas = sc_pp.vd_from_stats(self.gas_stats,
                                                      self.settings.PostProcess)
+        
+        # Then calculate correct concentrations for re-scaling the volume distributions:
+        nc, volume_concentration_oil, sample_volume, junge = sc_pp.nc_vc_from_stats(self.stats, self.settings.PostProcess,
+                                                                                    oilgas=sc_pp.outputPartType.oil)
+        self.vd_oil = (self.vd_oil / np.sum(self.vd_oil)) * volume_concentration_oil  # re-scale the volume distribution to the correct oil concentraiton
+        nc, volume_concentration_gas, sample_volume, junge = sc_pp.nc_vc_from_stats(self.stats, self.settings.PostProcess,
+                                                                                    oilgas=sc_pp.outputPartType.gas)
+        self.vd_gas = (self.vd_gas / np.sum(self.vd_gas)) * volume_concentration_gas  # re-scale the volume distribution to the correct gas concentraiton
```
Comment (tankefugl) [inline pysilcam/oilgas.py:158]: E501: line too long (153 > 130 characters)

File: pysilcam/oilgas.py [hunk 5]
```diff
@@ -142,10 +143,21 @@ def update(self):
...
+        # First calculate volume distributsion in units of volume concentration per sample volume
         self.dias, self.vd_oil = sc_pp.vd_from_stats(self.oil_stats,
                                                      self.settings.PostProcess)
         self.dias, self.vd_gas = sc_pp.vd_from_stats(self.gas_stats,
                                                      self.settings.PostProcess)
+        
+        # Then calculate correct concentrations for re-scaling the volume distributions:
+        nc, volume_concentration_oil, sample_volume, junge = sc_pp.nc_vc_from_stats(self.stats, self.settings.PostProcess,
+                                                                                    oilgas=sc_pp.outputPartType.oil)
+        self.vd_oil = (self.vd_oil / np.sum(self.vd_oil)) * volume_concentration_oil  # re-scale the volume distribution to the correct oil concentraiton
```
Comment (tankefugl) [inline pysilcam/oilgas.py:155]: E501: line too long (153 > 130 characters)

File: pysilcam/oilgas.py [hunk 6]
```diff
@@ -142,10 +143,21 @@ def update(self):
         self.gas_d50 = sc_pp.d50_from_stats(self.gas_stats,
                                             self.settings.PostProcess)
 
+        # First calculate volume distributsion in units of volume concentration per sample volume
         self.dias, self.vd_oil = sc_pp.vd_from_stats(self.oil_stats,
                                                      self.settings.PostProcess)
         self.dias, self.vd_gas = sc_pp.vd_from_stats(self.gas_stats,
                                                      self.settings.PostProcess)
+
```
Comment (tankefugl) [inline pysilcam/oilgas.py:151]: W293: blank line contains whitespace

**Review by tankefugl — CHANGES_REQUESTED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Review by tankefugl — DISMISSED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Review by WillRNaylor — COMMENTED** [review]
Just need to bump the version number.

**Review by WillRNaylor — APPROVED** [review]

**Pull request merged**